### DISCUSSION TITLE
how to Update a record ?

### DISCUSSION THREAD

**Discussion by haqamar** [khoih-prog/MySQL_MariaDB_Generic#15] [Q&A]
Hi its a very useful library but is there an example to update an existing record created earlier.?

**Comment by khoih-prog** [comment]
The library can't do anything **automagically**, but has to rely on the available MySQL / MariaDB functions.

To update a record, you have to use corresponding `UPDATE` function, as specified in [**UPDATE** Statement](https://dev.mysql.com/doc/refman/8.0/en/update.html).

Don't expect the examples will cover everything and can be applied directly to all use-cases. You must at least have to understand some basic knowledge of `MySQL / MariaDB functions`.

**Try any command in the terminal first to see everything is working correctly, then copy / adapt to your program.**
